website/cue/reference/components/sources/base/okta.cue

```diff
@@ -0,0 +1,136 @@
+package metadata
+
+base: components: sources: okta: configuration: {
+	domain: {
+		description: "The Okta subdomain to scrape"
+		required:    true
+		type: string: examples: ["foo.okta.com"]
+	}
+	scrape_interval_secs: {
+		description: """
+			The interval between scrapes. Requests are run concurrently so if a scrape takes longer
+			than the interval a new scrape will be started. This can take extra resources, set the timeout
+			to a value lower than the scrape interval to prevent this from happening.
```

Comment:
```suggestion
			The interval between scrapes. Requests are run concurrently so if a scrape takes longer
			than the interval, a new scrape will be started. This can take extra resources, set the timeout
			to a value lower than the scrape interval to prevent this from happening.
```